misc/staticbuildtestcontainer/getty-override.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
```

Comment:
Why do you have a blank ExecStart here?

Reply:
good question - copied from here ;)
https://github.com/aws-greengrass/aws-greengrass-lite/blob/c14fc1848c32084f971570d5e4a60ed632afad65/misc/container/getty-override.conf#L2